### PR TITLE
vpnbypass: support for procd_add_config_trigger

### PR DESCRIPTION
**Pull request by stangri**
Maintainer: me
Compile tested: ramips, ER-X, 19.07.5
Run tested: I don't have a spare router to test this as this conflicts with vpn-policy-routing

Description:
1. support for procd_add_config_trigger
2. updated README link
3. minor Makefile update

Depends on procd change:
This is still pending https://patchwork.ozlabs.org/project/openwrt/list/?series=218099

Signed-off-by: Stan Grishin <stangri@melmac.net>

### CODE REVIEW

File: net/vpnbypass/files/vpnbypass.init [hunk 1]
```diff
@@ -127,11 +132,15 @@ stop_service() {
...
-		procd_open_trigger
-			for n in $ifaces; do procd_add_reload_interface_trigger "$n"; procd_add_interface_trigger "interface.*" "$n" /etc/init.d/vpnbypass reload; done;
-			output "$serviceName monitoring interfaces: $ifaces\\n"
-		procd_close_trigger
+	local ifaces n
+	config_load network; config_foreach service_triggers_load_interface 'interface';
+	procd_open_trigger
+		procd_add_reload_trigger 'openvpn'
+		if type procd_add_service_trigger 1>/dev/null 2>&1; then
+			procd_add_service_trigger "service.restart" "firewall" /etc/init.d/${packageName} reload
```
Comment (feckert) [inline net/vpnbypass/files/vpnbypass.init:140]: As I know the `procd_add_service_trigger` is not merged in openwrt master!

Comment (stangri) [reply net/vpnbypass/files/vpnbypass.init:140]: Not to say that this should be merged before the dependent PR is merged, but with the code above the `procd_add_service_trigger` will be used in the init script as soon as it's available on the system. :)

**Comment by stangri** [comment]
This is ready to merge.

Also, please grant merge permission.

**Comment by feckert** [comment]
@stangri
I'm not sure, but I think the service trigger restart you are relying on is not yet merged? Or is it done differently now?

**Comment by stangri** [comment]
@feckert I've checked the ozlabs.org link and there's nothing there so I assumed it was merged. Either way, it checks if `procd_add_service_trigger` exists before invoking it, so I think it's safe to merge.

File: net/vpnbypass/Makefile [hunk 2]
```diff
@@ -5,7 +5,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=vpnbypass
 PKG_VERSION:=1.3.1
-PKG_RELEASE:=10
+PKG_RELEASE:=9
```
Comment (PolynomialDivision) [inline net/vpnbypass/Makefile:8]: you go from 10 to 9?

Comment (stangri) [reply net/vpnbypass/Makefile]: Ooops, my bad, don't know how I missed it.

**Comment by PolynomialDivision** [comment]
Still fails under x86_64?
```
Testing package vpnbypass in version 1.3.1 from vpnbypass
Installing vpnbypass (1.3.1-11) to root...
Collected errors:
 * satisfy_dependencies_for: Cannot satisfy the following dependencies for vpnbypass:
 * 	kernel (= 5.4.98-1-435e8cacd26efb34941107c904ecfdde)
 * opkg_install_cmd: Cannot install package vpnbypass.
Use package specific test.sh
Test failed
Error: Process completed with exit code 1.
```

**Comment by stangri** [comment]
> Still fails under x86_64?
> 
> ```
> Testing package vpnbypass in version 1.3.1 from vpnbypass
> Installing vpnbypass (1.3.1-11) to root...
> Collected errors:
>  * satisfy_dependencies_for: Cannot satisfy the following dependencies for vpnbypass:
>  * 	kernel (= 5.4.98-1-435e8cacd26efb34941107c904ecfdde)
>  * opkg_install_cmd: Cannot install package vpnbypass.
> Use package specific test.sh
> Test failed
> Error: Process completed with exit code 1.
> ```

That's evidently the problem with the build environment, not the package itself. I've poked around but I can't find the way to trigger the CI process myself, can you?

**Comment by stangri** [comment]
I'll resubmit another PR.

**Pull request closed without merging**